Tealium/src/ios/TealiumCordova.swift

```diff
@@ -94,6 +94,15 @@ class TealiumCordova: CDVPlugin {
         self.commandDelegate.send(result, callbackId: command.callbackId)
     }
 
+    @objc(gatherTrackData:)
+    public func gatherTrackData(_ command: CDVInvokedUrlCommand) {
+        TealiumPlugin.gatherTrackData(callbackId: command.callbackId)
+        
+        let result = CDVPluginResult(status: CDVCommandStatus_NO_RESULT)
+        result?.keepCallback = true
```

Comment:
don't think we need the `keepCallback` set to true. This is only for situations where the same callback is used repeatedly, which is not necessarily the case when calling the `gatherTrackData` method.